commit f5eb142cd04be2bc4ca610ed3b5b7e8ce3520ee3
Author: fanquake <fanquake@gmail.com>
Date:   Tue Jan 5 16:08:49 2021 +0800

    Don't invoke macOS SDK version checking

    This tries to use xcrun which is not available when cross-compiling.

diff --git a/qtbase/mkspecs/features/mac/default_post.prf b/qtbase/mkspecs/features/mac/default_post.prf
+++ b/qtbase/mkspecs/features/mac/default_post.prf
@@ -13,6 +13,4 @@
             QMAKE_XCODE_DEVELOPER_PATH \
             _QMAKE_STASH_ \
-
-        QMAKE_EXTRA_INCLUDES += $$shell_quote($$PWD/sdk.mk)
     }
